demisto_sdk/commands/download/tests/downloader_test.py

```diff
@@ -42,6 +42,8 @@ def __init__(self, tmp_path):
         self.SCRIPT_INSTANCE_PATH = None
         self.PLAYBOOK_INSTANCE_PATH = None
         self.LAYOUT_INSTANCE_PATH = None
+        self.PRE_PROCESS_RULES_INSTANCE_PATH = None
```

Comment:
How is this related?

Reply:
It was missing for `pre-processr-rule`, so I added it